rust/nasl-builtin-cryptographic/tests/hmac.rs

```diff
@@ -5,110 +5,54 @@
 #[cfg(test)]
 mod tests {
     use nasl_interpreter::*;
+    use test_utils::check_ok;
     #[test]
     fn hmac_md2() {
-        let code = r#"
-        HMAC_MD2(key: "my_shared?key", data: "so much wow");
-        "#;
-        let register = Register::default();
-        let binding = ContextFactory::default();
-        let context = binding.build(Default::default(), Default::default());
-        let mut parser = CodeInterpreter::new(code, register, &context);
-        assert_eq!(
-            parser.next(),
-            Some(Ok("9558b32badac84072d54422d05bd601a".into()))
+        check_ok(
+            r#"HMAC_MD2(key: "my_shared?key", data: "so much wow");"#,
+            "9558b32badac84072d54422d05bd601a",
         );
     }
 
     #[test]
     fn hmac_md5() {
-        let code = r#"
-        HMAC_MD5(key: "my_shared?key", data: "so much wow");
-        "#;
-        let register = Register::default();
-        let binding = ContextFactory::default();
-        let context = binding.build(Default::default(), Default::default());
-        let mut parser = CodeInterpreter::new(code, register, &context);
-        assert_eq!(
-            parser.next(),
-            Some(Ok("815292959633f0e63666d90d6f47cb79".into()))
+        check_ok(
```

Comment:
I think the name for the macro `check_ok` is a bit too vague as it does actually spin up a nasl-interpreter, runs the given code and verifies the Result, that's a bit more then just "check if output is Ok" which I would guess based on the name.

Reply:
Suggestion for a more descriptive name: `check_code_result`. Is that ok?